Makefile

```diff
@@ -325,6 +330,8 @@ ADDON_CONTROLLER = $(PWD)/governance-policy-addon-controller
 
 .PHONY: kind-bootstrap-hosted
 kind-bootstrap-hosted: kind-install-hosted install-crds install-resources kind-deploy-cert-manager setup-managedcluster
+	kubectl delete pod -l name=governance-policy-propagator -A --kubeconfig=./kubeconfig_hub
+	kubectl delete pod -l app=governance-policy-addon-controller -A --kubeconfig=./kubeconfig_hub
```

Comment:
why delete addon controller?

Reply:
I just needed to restart it. In my first test runs, it kept having problems when running in the workflow, but not when I was running it locally. When I added this, the tests started passing, I'm not 100% sure why 😆

Reply:
ok!! but so weird I haven't seen that 
oh maybe  `PLACEMENT_TAG=latest-amd64 make kind-bootstrap-hosted ` Are you mac?